settings.gradle

```
@@ -8,4 +8,5 @@
  */
 
 rootProject.name = 'openapi-boot-starter'
-include 'starter'
\ No newline at end of file
+include 'starter'
+include 'starter-test'
```

```suggestion
include 'example'
```

please name it like that, so it would be clear that its a separate project to apply the lib as an example